src/main/java/picard/analysis/TheoreticalSensitivity.java

```diff
@@ -354,9 +359,13 @@ public static double theoreticalSensitivity(final Histogram<Integer> depthHistog
      * @return Distribution of base qualities removing any trailing zeros
      */
      static double[] trimDistribution(final double[] distribution) {
-         int endOfDistribution = distribution.length - 1;
-         while(distribution[endOfDistribution] == 0) {
+         int endOfDistribution = distribution.length;
+         if (distribution.length == 0) {
+             return distribution;
+         }
+         while(distribution[endOfDistribution - 1] == 0) {
```

Comment:
what if the distribution is empty (as in all the entries are zero) should you protect against that as well?

Reply:
I believe it already handled the case you describe, but I added some test cases to cover it and modified the comment to hopefully better describe what is happening.